### ISSUE TITLE
Library Scan Triggered Automatically while Scanner Disabled

### ISSUE THREAD

**Issue by dalkain** [Unmanic/unmanic#232]
### Description
Overnight, a full library scan was somehow triggered even though all library scanner options are disabled in my container. This scan triggered while I was sleeping according to the logs, but there is nothing to indicate why a library scan was triggered.
`2021-10-01T07:03:24:INFO:Unmanic.LibraryScannerManager - [FORMATTED] - Running full library scan`

### Steps to Reproduce
My only theory is that, similar to https://github.com/Unmanic/unmanic/issues/224, the setting for "Library scan schedule in minutes" is still taking effect even though it is hidden by "Enable the library scanner" being disabled. My container had been running for just over a day when the scan started and the default value for this setting is 1440 minutes (24 hours).

### Expected behavior:
If "Enable the library scanner" is disabled in the settings, the only way to trigger a full library scan should be if I manually start it from the Pending Tasks section of the dashboard

### Versions
Version - 0.1.1+68717c4

### Installation Method Used
Docker on Unraid 6.9.2 using the josh5/unmanic repo

### Additional Information
Library settings when this scan triggered:
`"enable_inotify": false,`
`"enable_library_scanner": false,`
`"run_full_scan_on_start": false,`
`"schedule_full_scan_minutes": "1440",`

**Comment by dalkain** [comment]
Since I accidentally overwrote my container when trying to deploy a second one to test now that I'm off work...I figured I'd just test things now and confirm and provide more details on my findings.

The behavior matches my theory in "Steps to Reproduce" above.

Started the container with the following configs:
```
    "enable_inotify": false,
    "enable_library_scanner": false,
    "follow_symlinks": false,
    "run_full_scan_on_start": false,
    "schedule_full_scan_minutes": "1",
```
During startup, the following appeared in the logs (omitting non-LibraryScannerManager lines):
```
2021-10-01T18:12:57:INFO:Unmanic - Starting LibraryScannerManager
2021-10-01T18:12:57:INFO:Unmanic.LibraryScannerManager - [FORMATTED] - Starting LibraryScanner Monitor loop
2021-10-01T18:12:57:INFO:Unmanic.LibraryScannerManager - [FORMATTED] - Setting LibraryScanner schedule to scan every 1 mins...
2021-10-01T18:13:59:INFO:Unmanic.LibraryScannerManager - [FORMATTED] - Running full library scan
```

Also, changing the `Library scan schedule in minutes` value **will not** take effect on submit unless you also leave `Enable the library scanner` enabled when submitting. It will still update the config file, but the new value won't take effect or show in the logs as being changed.

It seems to me like the LibraryScannerManager isn't actually checking to see if the library scanner is set to 'Enabled' before running automated scans or changing the store config file, but only worrying about it being enabled to make changes to the library schedule while the container is running (I don't know if the other Library settings make any announcement in the logs; I've never seen them show up there)

**Comment by dalkain** [comment]
PSA for anyone that sees this issue and also doesn't want the library to auto-scan on an interval while this issue is present: Do not set the scan timer to a value above 4 billion.

The application will accept as large of values as you put in and submit while running, but if it is to large it will throw errors in the background on container startup when it attempts to pull in the LibraryScannerManager configuration. The value is limited by two things: C int conversion (max of 4,294,967,295) and more importantly in this case the maximum allowed time from python's datetime (which is the year 9999, or a value somewhere between 4-4.2b minutes from now).

**Comment by Josh5** [comment]
Resolved in commit 454c92c
